content/actions/use-cases-and-examples/building-and-testing/index.md

```diff
@@ -27,6 +27,7 @@ children:
   - /building-and-testing-powershell
   - /building-and-testing-python
   - /building-and-testing-ruby
+  - /building-and-testing-rust
```

Comment:
somehow this doesn't show up: https://docs-35725-22a95e.preview.ghdocs.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust